src/com/epam/javacourse/homework2/Palindrome.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.javacourse.homework2;
+
+public class Palindrome {
+    public boolean isPalindrome(int x) {
+        if (x < 0) {
+            return false;
+        }
+        int number = x;
+        int rev = 0;
```

Comment:
It's always better to initialize variables at the beginning of the method, it makes code more readable.
If you want to initialize them only if `x>0` then let's change the condition on line 5 to `x>0`.
It will help to make it a bit prettier.

Reply:
Fixed.